script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -979,7 +979,7 @@ Her heavy breathing rises from my neck all the way to my brain.
 [sha:38748bd814a07132f92b714ce0137a30a57c354c]{
 -- Page 615, Offset 25900.
 -- それは駆け巡る脳内物質のように、彼女の昂ぶりを伝えてくる。
-It's like a neurotransmitter firing to convey her arousal.
+Her arousal courses like a chemical surging across my brain.
```

Comment:
Yeah tbh this sentence is a bit weird, but not sure this is correct. I think the main thrust of this sentence is the 伝える, which is a bit lost here. Maybe run this one by Petrikow after all.

Reply:
So yeah, I think that a literal tl of this is something along the lines of
> Like a rushing intracranial substance, it [her breathing] communicates her arousal [to me].

I think I wanted to lean away from repeating `brain` since it's in the prev line, but you're right that maybe we can get closer to the SL here. Something like
> It's like a chemical message carrying her arousal straight to my grey matter.
> It's like a chemical message that floods my head with her arousal.

avoids the brain ref, I feel like there's gotta be a better bio term as well than 'brain chemical' but endorphin or hormone don't super fit, and a more generic term like neuropeptide is a bit science-y (hence neurotransmitter, though I agree it's imperfect). Let me know your thoughts.

Reply:
Part of me wants to try to keep 'brain' and the sentence pattern of the first line for repetition's sake but realistically that's the least of the problems. 

I think: "It's like a chemical message that floods my head with her arousal." may be the best bet...

Reply:
this suggestion works for me:
> It's like a chemical message that floods my head with her arousal.